libs/styles/styles.css

```diff
@@ -263,8 +263,8 @@
 }
 
 .body-xxs {
-  font-size: var(--type-heading-xxs-size);
-  line-height: var(--type-body-xs-lh);
+  font-size: var(--type-body-xxs-size);
+  line-height: var(--type-body-xxs-lh);
 }
```

Comment:
This is the misconfiguration fix?

Reply:
@rgclayton yes, now that I am using the typography styles for legal variant in text I noticed these were wrong